### DISCUSSION TITLE
--alignIntronMax parameters  setting

### DISCUSSION THREAD

**Discussion by DiracZhu1998** [alexdobin/STAR#1329] [General]
Dear contributors, 
Thank you for giving us a wonderful RNA alignment tools, STAR, hooray!
I have a question about default --alignIntronMax parameters (2ˆwinBinNbits)*winAnchorDistNbins), 
how do you set the default value of winBinNbits and winAnchorDistNbins? 
What dataset you based on to conclude?
If I calculate a intron distribution and choose 99 percentile as --alignIntronMax parameter (max gap open length I summarize) is OK  or not ?
The intron distribution file attached, you may want to check.
Thank you for your help!
[Intron_distribution_log10.pdf](https://github.com/alexdobin/STAR/files/7005758/Intron_distribution_log10.pdf)

**Comment by alexdobin** [comment]
Hi @DiracZhu1998 

you can specify directly the value of ```--alignIntronMax```, the other parameters ```--winBinNbits --winAnchorDistNbins``` are only used if you **do not** specify this value.
I would go with the maximum intron length annotated in the species of interest, unless it's outrageously large - which does not seem to be the case judging by your distribution.

Cheers
Alex

**Reply by DiracZhu1998** [reply]
Great! Thanks!
